kit/transport/http/middleware.go

```diff
@@ -53,7 +53,11 @@ func Metrics(name string, reqMetric *prometheus.CounterVec, durMetric *prometheu
 
 func SkipOptions(next http.Handler) http.Handler {
 	fn := func(w http.ResponseWriter, r *http.Request) {
-		if r.Method == "OPTIONS" {
+		// Preflight CORS requests from the browser will send an options request,
+		// so we need to make sure we satisfy them
+		origin := r.Header.Get(("Origin"))
+
+		if origin == "" && r.Method == "OPTIONS" {
 			return
```

Comment:
I know we just return here, but I wonder if we're better off returning a different status code than `200`?